common/conf/env/DEV.properties

```diff
@@ -86,3 +86,6 @@ admin.oauth.callback=http://localhost:9000/oauth2callback
 #Preview Oauth
 preview.oauth.clientid=768784040766-1eq1drssgu65vrs4oi1hjh70d9b30fti.apps.googleusercontent.com
 preview.oauth.callback=http://localhost:9000/oauth2callback
+
+# Beacon diagnostics
+beacon.url=//localhost:9000
```

Comment:
I wasn't too bothered about cross-domain, just wanted it to work in dev. if you're testing cross-domain, you probably need to think about what your client and resource domains should be during testing.